sdk/identity/azure-identity/README.md

```diff
@@ -1,17 +1,6 @@
 # Azure Identity client library for Java
 The Azure Identity library provides Azure Active Directory token authentication support across the Azure SDK. It provides a set of TokenCredential implementations which can be used to construct Azure SDK clients which support AAD token authentication.
 
- This library currently supports:
-  - [Service principal authentication](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals)
-  - [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview)
-  - [Device code authentication](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-device-code)
-  - Interactive browser authentication, based on [OAuth2 authentication code](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow)
-  - [Username + password authentication](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth-ropc)
-  - IntelliJ authentication, with the login information saved in Azure Toolkit for IntelliJ
-  - Visual Studio Code authentication, with the login information saved in Azure plugin for Visual Studio Code
-  - Azure CLI authentication, with the login information saved in Azure CLI
-  - Shared Token Cache credential, which shares login information with Visual Studio, Azure CLI, and more
-
```

Comment:
Any reason for removing this? This was a good reference for finding the list of supported auth.

Reply:
we moved these down to the table in Cred classes section.